cmd/mongo/binary_backup_fetch.go

```diff
@@ -0,0 +1,45 @@
+package mongo
+
+import (
+	"context"
+	"os"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal/databases/mongo/binary"
+	"github.com/wal-g/wal-g/utility"
+)
+
+const binaryBackupFetchCommandName = "binary-backup-fetch"
+
+var binaryBackupFetchCmd = &cobra.Command{
+	Use:   binaryBackupFetchCommandName + " <backup name> <replSetName> <mongod dbpath> <mongod version>",
+	Short: "Fetches a mongodb binary backup from storage and restores it in mongodb storage dbPath",
+	Args:  cobra.ExactArgs(4),
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Too much logic for the command Run function, I guess you might want to move it to a corresponding handler in the internal/databases/mongo.